packages/crypto/src/strategies/SodiumBip32Ed25519.ts

```diff
@@ -12,77 +12,79 @@ import {
 } from '../hexTypes';
 import { Ed25519PrivateKey, Ed25519PublicKey, Ed25519Signature } from '../Ed25519e';
 import { HexBlob } from '@cardano-sdk/util';
+import sodium from 'libsodium-wrappers-sumo';
 
 const EXTENDED_KEY_HEX_LENGTH = 128;
 
 export class SodiumBip32Ed25519 implements Bip32Ed25519 {
+  // Prevent instantiation
+  private constructor() {
+    // Empty
+  }
```

Comment:
Why do we need to export `ready` from index then? Wouldn't it always be initialized as the only way to create is the `async create`?

Reply:
There are two ways to use the package, one is to use the strategy, in that case the factory method will take care of the initialization, however, you can use the types directly without the strategy instance, in that case, you must make use its initialized before using those. I mean types such as `Ed25519PrivateKey`